test/config_test.py

```diff
@@ -1,14 +1,14 @@
 import pytest
 
-from oci import config
+from ovirt_ci import config
 
 
 def test_load_minimal(tmpdir):
     cfg_data = """[jenkins]
 user_id = fred
 api_token = 12345
 """
-    cfg_file = tmpdir.join("oci.conf")
+    cfg_file = tmpdir.join("ovirt_ci.conf")
```

Comment:
ovirt-ci.conf

(same for all other conf file names in this file)

Reply:
Done